packages/components/src/components/post-list/post-list.tsx

```diff
@@ -0,0 +1,75 @@
+import { Component, Element, Prop, Host, State, h } from '@stencil/core';
+import { version } from '@root/package.json';
+
+/**
+ * @slot default - Slot for placing the list title
+ * @slot post-list-item - Slot for placing post-list-item components
+ */
+
+@Component({
+  tag: 'post-list',
+  styleUrl: 'post-list.scss',
+  shadow: false,
+})
+export class PostList {
+  @Element() host: HTMLPostListElement;
+
+  /**
+   * The unique title of the list that is also referenced in the labelledby
+   */
+  @State() uuid: string;
+
+  /**
+   * The list title can be hidden be setting `title-hidden="true"` or just `title-hidden`
+   */
+  @Prop() readonly titleHidden: boolean = false;
+
+  /**
+   * The list can become horizontal by setting `horizontal="true"` or just `horizontal`
+   */
+  @Prop() readonly horizontal: boolean = false;
+
+  /**
+   * Define the gap of the list items using the --post-list-item-gap custom property.
+   * e.g. --post-list-item-gap: 1rem 0.5rem;
+   */
+
+  /**
+   * Define the gap between the title/heading and the list items using the --post-list-heading-gap custom property.
+   * e.g. --post-list-heading-gap: 2rem;
+   */
```

Comment:
Can you remove these comments and add the information to the documentation instead please?